pkg/gadgets/trace/capabilities/tracer/bpf/capable.h

```diff
@@ -9,6 +9,9 @@
 
 struct cap_event {
 	__u64	mntnsid;
+	__u64	current_userns;
+	__u64	target_userns;
+	__u64   cap_effective;
```

Comment:
It's defined as u32 in the kernel, any reason to use a different width?

Reply:
I can change the *_userns fields to u32. But the cap_effective needs to stay 8 bytes.